modules/ROOT/pages/index.adoc

```diff
@@ -26,6 +26,30 @@ The following table provides component information for Anypoint Platform PCE:
 | API Testing	|   | X |
 |===
 
+For Anypoint Platform PCE Version 2.1, the following policies are included with API Manager:
```

Comment:
The versions for the polices need to be specified. Tho I'm not sure ATM where to take them from.
Where did you get this info from?

Reply:
Info in Slack: https://github.com/mulesoft/docs-private-cloud/pull/141